### PR TITLE
Add gnome-41

### PR DESCRIPTION
**Pull request by andyholmes**
Add the latest GNOME Platform and SDK

### CODE REVIEW

**Comment by andyholmes** [comment]
I'm not sure exactly how the docker build works here, so maybe we need to wait for Fedora 35. Feel free to just take this over and make any changes necessary.

File: .github/workflows/flatpak-test.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ jobs:
     name: Flatpak Builder
     runs-on: ubuntu-latest
     container:
-      image: bilelmoussaoui/flatpak-github-actions:gnome-40
+      image: bilelmoussaoui/flatpak-github-actions:gnome-41
```
Comment (bilelmoussaoui) [inline .github/workflows/flatpak-test.yml:12]: This won't work for now. The job has to run for the image to exists

**Comment by bilelmoussaoui** [comment]
Looks good to me, we can provide 41 for now and bump to F35 later.

**Comment by bilelmoussaoui** [comment]
Thanks

**Pull request merged**